### DISCUSSION TITLE
Is there any opportunity in this community to Test / Automate any newly/existing develop feature

### DISCUSSION THREAD

**Discussion by abhinay5993** [Your-First-Open-Source-Project/Your-First-Open-Source-Project#567] [General]
Hello @Infinity-chakra  / @melvincwng
Is there any opportunity in this community to **Test / Automate** any _newly/existing_ develop feature.
Thanks!! in Advance. Will wait for your reply.
Lets!! have grate collaborative discussions using the **gitHub** discussion feature.

**Comment by melvincwng** [comment]
Hi @abhinay5993, thank you for your interest

Currently we do not have any opportunities for automation/testing

Will update if future opportunities arise
